### PR TITLE
Modified wholeBodyDynamics config for standup demo simulation

### PR DESCRIPTION
**Pull request by gabrielenava**
Edited wholeBodyDynamics .xml files for simulations. Added hands ports, and other modifications for running wholeBodyDynamics during standup demo.

### CODE REVIEW

File: src/devices/wholeBodyDynamics/app/wholebodydynamics-icub-external-six-fts-sim.xml [hunk 1]
```diff
@@ -34,6 +38,8 @@
         </group>
 
         <group name="WBD_OUTPUT_EXTERNAL_WRENCH_PORTS">
+            <param name="/wholeBodyDynamics/left_arm/endEffectorWrench:o">(l_hand,l_hand_dh_frame,l_sole)</param>
```
Comment (traversaro) [inline src/devices/wholeBodyDynamics/app/wholebodydynamics-icub-external-six-fts-sim.xml]: Why are you expressing this force with the orientation of the `l_sole` ? This configuration file is supposed to be general, i.e. working in a meaningful way even when the `l_sole` frame is not attached to the ground.

Comment (gabrielenava) [reply src/devices/wholeBodyDynamics/app/wholebodydynamics-icub-external-six-fts-sim.xml]: I copy-pasted ports definitions from `iCubGenova02` file (wholeBodyDynamics.xml`). Should I just remove `l_sole` and `r_sole`?

**Comment by gabrielenava** [comment]
Done with https://github.com/robotology/codyco-modules/pull/245/commits/3d7cf166366faf09c27b2a59332fb3da58783472

**Comment by traversaro** [comment]
Great, the two parameters version should work fine. Can you propagate this changes to all the configuration files, so we keep consistency?

**Comment by gabrielenava** [comment]
I'm not sure about propagating these changes, because of [this issue](https://github.com/robotology-playground/WBI-Toolbox-controllers/issues/56). Maybe it is better to investigate first why the real robot has this behaviour before changing WBD configuration.

**Comment by traversaro** [comment]
@gabrielenava What is the status of this?

**Comment by gabrielenava** [comment]
The blocking issue has been recently solved, but in the end it turns out that for the real robot we still need two different configurations (one for yoga and the other for standup), and we cannot merge the two configurations together as for the simulation.

**Comment by traversaro** [comment]
Ok, so in the meanwhile we just merge this PR?

**Comment by gabrielenava** [comment]
Yes, for me we can merge this PR.

**Comment by traversaro** [comment]
@gabrielenava Sorry gabri, I forgot to merge this branch, can we merge it now?

**Comment by gabrielenava** [comment]
in theory yes, but I saw now that not all travis test have been passed. Can we merge anyway?

**Comment by traversaro** [comment]
Yes, it is a problem that was solved by https://github.com/robotology/codyco-modules/pull/251 but is still present in this old branch.

**Pull request merged**